noxfile.py

```diff
@@ -2,17 +2,24 @@
 
 
 @session(python=["3.5", "3.6", "3.7", "3.8"], reuse_venv=True)
-def test(session):
+def configuration(session):
     session.install(".")
     session.install("-r", "requirements-test.txt")
 
-    if session.posargs is None:
-        session.run("pytest", "tests")
-    else:
+    if session.posargs:
         session.run("pytest", *session.posargs)
+    else:
+        session.run("pytest", "-k", "TestConfiguration")
+
+
+@session(python=["3.8"], reuse_venv=True)
+def example(session):
+    session.install(".")
+    session.install("-r", "requirements-test.txt")
+    session.run("pytest", "-k", "test_example", "-s")
```

Comment:
this appears to be inconsistent w/ `TestConfiguration` below, would be good to remain consistent

Reply:
How is this inconsistent? These example tests are not intended to be run as part of CI, if that helps explain things.